bumpversion/vcs.py

```diff
@@ -16,11 +16,13 @@
 
 class BaseVCS(object):
     @classmethod
-    def commit(cls, message):
+    def commit(cls, message, context):
         with NamedTemporaryFile("wb", delete=False) as f:
             f.write(message.encode("utf-8"))
         env = os.environ.copy()
         env[str("HGENCODING")] = str("utf-8")
+        for key in ["current_version", "new_version"]:
+            env[str("BUMPVERSION_"+key.upper())] = context[key]
```

Comment:
I'm wondering if the value should also be explicitly casted to a `str` since we do the same for others.

Reply:
Added the str cast since certainly makes things more consistent. I also added this feature to the readme so that users can be made aware of it.